src/types.ts

```diff
@@ -8,3 +8,7 @@ export interface AppsPluginOptions {
   serviceMethod: ServiceMethod;
   thumbnailsPrefix: string;
 }
+
+// todo: get from plugin-file, currently the enum is defined as integer
```

Comment:
Is there an issue related to that ?

Reply:
Yes because we cannot use the value of the enum to set it to another place. It would simplify the use if it was the case.

Reply:
Ok, but my question is: is there an issue opened on github related to this todo, so we don't forget to tackle it at some point?